lib/registerTypes/RegisterTypes.h

```diff
@@ -388,6 +388,7 @@ void registerTypes(Serializer &s)
 	registerTypesClientPacks2(s);
 	registerTypesServerPacks(s);
 	registerTypesLobbyPacks(s);
+	s.template registerType<ILimiter, OppositeSideLimiter>();

```

Comment:
This should be before all NetPacks (in `registerTypesMapObjectTypes`)

Reply:
Ok